sdk/src/main/java/com/qonversion/android/sdk/Qonversion.kt

```diff
@@ -159,4 +160,22 @@ interface Qonversion {
      * for the most part of use cases.
      */
     fun finish()
+
+    /**
+     * Get the most recent available User info.
+     * @return [User] instance.
+     */
+    @Throws(QonversionException::class)
+    suspend fun getUserInfo(): User
+
+    /**
+     * For the function description see [getUserInfo].
+     * @param onSuccess returns [User] instance.
```

Comment:
We should name it a callback, I guess, like `a callback to be called when the user info is received` or smth like that. 
And the below one - `a callback to be called on error`...